### ISSUE TITLE
kebabCase can leave trailing hyphen

### ISSUE THREAD

**Issue by patrickcate** [angus-c/just#442]
I noticed that the `kebabCase` function leaves a trailing hyphen if the ending character is not `A-Za-z`. Lodash and other string case libraries do not include trailing hyphens.

```javascript
// Just
kebabCase('Rank (Ordinal)'); // => rank-ordinal-

// Lodash
_.kebabCase('Rank (Ordinal)'); // => rank-ordinal
```

**Comment by angus-c** [comment]
Thank you. I will fix this

**Comment by angus-c** [comment]
Fixed in just-kebab-case@4.0.2
